lib/puma/launcher.rb

```diff
@@ -87,6 +87,8 @@ def initialize(conf, launcher_args={})
       Puma.stats_object = @runner
 
       @status = :run
+
+      log_config if ENV['LOG_CONFIG']
```

Comment:
@MSP-Greg is right - any new ENV variables should be prefixed with "PUMA_"

Reply:
Makes sense 👍